week-3/A-array-find/exercise.js

```diff
@@ -5,7 +5,9 @@
 */
 
 // -- Complete this function -->
-function findLongNameThatStartsWithA(names) {}
+function findLongNameThatStartsWithA(names) {
```

Comment:
This is good and correct. You need to be a little careful of merge conflicts with doing week-3 stuff in the week-4 pull request though.